packages/loopring_v3/contracts/aux/bridge/BatchDepositor.sol

```diff
@@ -55,7 +55,7 @@ abstract contract BatchDepositor is IBatchDepositor, ReentrancyGuard
     uint32             public immutable accountID;
     IDepositContract   public immutable depositContract;
 
-    mapping (uint => mapping (bytes32 => uint)) public pendingDeposits;
+    mapping (bytes32 => uint)                   public pendingTransferBatches;
```

Comment:
I thought this pendingTransferBatches only represents unhandled outbound L2 transfers (bridge -> users), but now I'm not sure...

Reply:
That's correct. These are pending deposits, so `batchDeposit` already moved the funds to the Bridge account on L2, and these transfers are for the distribution of these deposits on L2.

The source of these deposits could be 
- An external call to `batchDeposit`
- The deposits needed as returned by connectors
- If the connector call failed these will also be used to return funds to the users on L2

In all cases they are pending deposits needing to be processed on L2, so `pendingDeposits` seems like a good name to me, `pendingTransferBatches` more implementation specific so a bit less clear I think what it does using just the name.